modules/build_update_subfolders.nf

```diff
@@ -0,0 +1,18 @@
+// builds or updates the subfolders with log and predictions files
+process BUILD_UPDATE_SUBFOLDERS {
+    secret "SYNAPSE_AUTH_TOKEN"
+    container "sagebionetworks/synapsepythonclient:v2.7.0"
+
+    input:
+    tuple val(submission_id), val(build_or_update)
+    val project_name
+    val ready
+
+    output:
+    val "ready"
```

Comment:
I would fine a way to output the two synapse folder ids that were created.

Reply:
Hi Tom, yes I can follow the logic written for `SCORE`s output. I'm guessing you're suggesting this for the future when these subfolders need to get updated with the output `predictions.csv` files?

Reply:
Yup. that's right!

Reply:
Sounds good - since this folds into the scope of [my next ticket](https://sagebionetworks.jira.com/browse/IBCDPE-809), I'll use your suggestions for the corresponding PR of that one, and leave it out of this PR. Thanks!